package.json

```diff
@@ -8,8 +8,8 @@
     "start": "next start",
     "test": "NODE_ENV=test && jest --watch",
     "test:ci": "jest --ci --passWithNoTests",
-    "local": "docker-compose up -d && npx prisma studio",
-    "stop-local": "docker-compose down",
+    "local": "docker compose up -d && npx prisma studio",
```

Comment:
Needed to change due to docker version update. Link: https://stackoverflow.com/questions/78769459/command-not-found-docker-compose-after-docker-4-32-0-update